sqlglot/parser.py

```diff
@@ -6671,12 +6677,15 @@ def _parse_credentials(self) -> t.Optional[exp.Credentials]:
         expr = self.expression(exp.Credentials)
 
         if self._match_text_seq("STORAGE_INTEGRATION", advance=False):
-            expr.set("storage", self._parse_conjunction())
+            expr.set("storage", self._parse_property(parse_as_var=True))
```

Comment:
Why was this changed? The spec shows:

```
STORAGE_INTEGRATION = <integration_name>
```

This indicates that the right-hand side should be an `Identifier`, not a `Var`.